week-2/13-order/readme.md

```diff
@@ -0,0 +1,13 @@
+# Item order
+
+The `flex-order` property is used to move a *specific* item to a different position in the list.
+
+> **Remember:** if you forgot how to use the `flex-order` property, you can always:
```

Comment:
I've been putting hints after the description of the exercise. I feel like if a student opens an exercise they'll want to know what they have to do first? Either way, let's choose an order and be consistent.

Reply:
ok, I'll move it after 👍